client/layout/guided-tours/docs/API.md

```diff
@@ -0,0 +1,223 @@
+# Guided Tours API
+
+Guided Tours are declared in JSX as a tree of elements. All of them are available as named exports from `client/layout/guided-tours/config-elements`.
+
+## All-in example
+
+Before we go into full details, let's look at this example. It includes most possible things you can use.
+
+<img src="https://cldup.com/QBoRhc-P0M.png" width="515" />
+
+Here is the code used:
+
+```
+<Step name="example" placement="below" target="my-sites" arrow="top-left">
+  <p>Plain text description.</p>
+  <p>Multiple lines.</p>
+  <Continue step="next-step" click target="my-sites" icon="my-sites" />
+  <ButtonRow>
+    <Next step="next-step" />
+    <Quit />
+  </ButtonRow>
+  <Link href="https://learn.wordpress.com">
+    { translate( 'Learn more about WordPress.com' ) }
+  </Link>
+</Step>
+```
+
+## Tour
+
+Tour is a React component that declares the top-level of a tour. It defines conditions for starting a tour and contains `<Step>` elements as children.
+
+### Props
+
+* `name`: (string) Unique name of tour in camelCase.
+* `version` (string): Version identifier. We use date string like "20161224".
+* `path` (string or array, optional): Use this prop to limit tour only to some path prefix (or more prefixes if array). Example: `[ '/stats', '/settings' ]`
+* `when` (function, optional): This is a redux selector function. Use this to define conditions for the tour to start.
+* `children` (nodes): only supported type is `<Step>`
+
+## Step
+
+Step is a React component that defines a single Step of a tour. It is represented as dark box on top of Calypso UI. Step can be positioned in many ways in relation to any DOM node in Calypso that is marked with `[data-tip-target]` attribute.
```

Comment:
ah, you say it below. maybe remove implementation details here and just say "DOM node in Calypso".